test/FsAutoComplete.Tests.Lsp/ExtensionsTests.fs

```diff
@@ -406,6 +406,6 @@ let signatureTests state =
     "signature evaluation"
     [ testList
         "tests"
-        [ verifySignature 0 (4, 16) "val arrayOfTuples: (int * int)[]"
+        [ verifySignature 0 (4, 16) "val arrayOfTuples: (int * int) array"
```

Comment:
the default output for array types has changed from `[]` to `array` like other container types.